microprofile/fault-tolerance-smallrye/extension/src/main/java/org/wildfly/extension/microprofile/faulttolerance/deployment/MicroProfileFaultToleranceDeploymentProcessor.java

```diff
@@ -70,6 +70,8 @@ public void deploy(DeploymentPhaseContext phaseContext) {
 
         if (registeredSubsystems.contains("microprofile-metrics-smallrye")) {
             metricsIntegration = MetricsIntegration.MICROPROFILE_METRICS;
+        } else if (registeredSubsystems.contains("micrometer")) {
+            metricsIntegration = MetricsIntegration.MICROMETER;
```

Comment:
I'm not sure you really need to check for MP Metrics here, as it is no longer part of the system, and any older configurations that did have it will be forced to migrate and remove it. Regardless, if you'd like to keep it for safety's sake or whatever, I would put Micrometer first, as you're more likely to find that.